src/sagemaker/xgboost/defaults.py

```diff
@@ -14,7 +14,9 @@
 from __future__ import absolute_import
 
 XGBOOST_NAME = "xgboost"
+XGBOOST_VERSION_0 = "0.90"
```

Comment:
The naming scheme of `XGBOOST_VERSION_0`, `XGBOOST_VERSION_1`, `XGBOOST_VERSION_2`, isn't very descriptive or helpful, and `XGBOOST_VERSION_0` is misleading, as someone reading the code without context would think it points to a different image compared to `XGBOOST_VERSION_1`. Either let's think of a better name for `XGBOOST_VERSION_0` and `XGBOOST_VERSION_1`, or let's not add this constant.